contents/blog/customer-support-at-posthog.md

```diff
@@ -0,0 +1,34 @@
+---
+date: 2021-05-17
+title: How We Do Customer Support At Our Open-Source DevTool Company
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+By: [Mohammed Shehu](https://twitter.com/shehuphd), Content Lead @ PostHog
+
+Customer support is a critical function of any startup, especially if you offer a highly technical product like we do. [Charles Cook](https://www.linkedin.com/in/charlescook1/), Business Head at PostHog, shared his thoughts on how CS was carried out at his previous companies (VITL and Roli) and how customer support has evolved here at PostHog over the past few months. 
+
+[Roli](http://www.roli.com/) sells musical instruments and production software. Because the company deals with creators, the customer support team consists of qualified technical musicians, producers, and composers in their own right. Reflecting the makeup of your customer base is key to ensuring your CS team can relate to the problems that your customers face and provide better, more effective solutions.
```

Comment:
going into this much depth this early about companies from other industries is a turn off - maybe we should put a title above ie "What do devtools, musical instruments and nutrition have in common?"

Reply:
Yeah I didn't mind talking about it in the short interview as it was in response to specific questions and made more sense, but personally I would prefer to not talk about my previous companies in the article to this extent - could just say something in summary like 'Drawing on his experiences at ROLI and Vitl, he thought about how we could take customer support to the next level.'